tests/Fable.Tests/Fable.Tests.fsproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="Tests.fs" />
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="../../src/Hedgehog/Hedgehog.fsproj" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Fable.Core" Version="3.1.5" />
```

Comment:
That reference (`Fable.Core`), `webpack.config.js` and `package.json` are what enable Fable, right?

Reply:
I am not sure if this is all we need. Here is a minimal fable example: https://github.com/fable-compiler/fable2-samples/tree/master/minimal

Reply:
But as we are not going to run it in browser I removed react and react-dom and webpack-dev-server.

Reply:
As we are not running the Hedgeho in browser we could maybe use this as template: https://github.com/fable-compiler/fable2-samples/tree/master/nodejs

Reply:
But to answer you question 😄. Yes Fable.Core reference, webpack.config.js and package.json is what enables fable.

Reply:
Yep, I'm familiar with all those except Fable :sweat_smile: But the idea of making this Fable-friendly is awesome. I put up some time for this week to look around and see how this all works.